src/app/data-model-provider/Provider.h

```diff
@@ -57,6 +59,15 @@ class Provider : public ProviderMetadataTree
     // event emitting, path marking and other operations
     virtual InteractionModelContext CurrentContext() const { return mContext; }
 
+    /// Validates that the given event path is supported, where path may contain wildcards.
```

Comment:
That's not what it validates.  It validates that expanding the event path includes at least one concrete event path that the subject descriptor has access to, in theory.

An event path can be perfectly "valid" but not accessible by the subject.

Reply:
True ... need to work more on naming and description ... will iterate on this.